src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends ColorSupplier {
+    public Ball getRandomBall() {
+        Random random = new Random();
+        Ball ball = new Ball(getRandomColor(), random.nextInt(100));
+        return ball;
```

Comment:
we can simplify this, but it is optional:
```suggestion
        return new Ball(getRandomColor(), random.nextInt(100));
```